integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/clients/GreenMailClient.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.esb.integration.common.utils.clients;
+
+import com.icegreen.greenmail.user.GreenMailUser;
+import com.icegreen.greenmail.user.UserException;
+
+import java.util.Date;
+import java.util.Map;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * Client class for GreenMail Server
+ */
+public class GreenMailClient {
+    private GreenMailUser greenMailUser;
+
+    public GreenMailClient(GreenMailUser greenMailUser){
+        this.greenMailUser = greenMailUser;
+    }
+
+    /**
+     * Sending email to the user account in the server
+     * @param subject Email subject
+     * @throws MessagingException
+     * @throws UserException
+     * @throws InterruptedException
+     */
+    public void sendMail(String subject) throws MessagingException, UserException, InterruptedException {
+        MimeMessage message = new MimeMessage((Session) null);
+        message.setFrom(new InternetAddress("some@mail.com"));
```

Comment:
Shall we define constants to the email addresses(strings) that are repititively used?

Reply:
done